packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -507,7 +508,7 @@ function commitHookEffectListUnmount(
   }
 }
 
-function commitHookEffectListMount(tag: number, finishedWork: Fiber) {
+function commitHookEffectListMount(tag: HookFlags, finishedWork: Fiber) {
```

Comment:
Not sure about changing this type to `HookFlags` since its technically the bitwise or of flags